src/test/java/BaseTest.java

```diff
@@ -1,12 +1,59 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 
+import java.time.Duration;
 
-public class BaseTest {
 
+public class BaseTest {
+    WebDriver driver;
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
 
+    @BeforeMethod
+    public void setUpBrowser() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
```

Comment:
How to handle chrome notification in Selenium

![image](https://user-images.githubusercontent.com/104761482/222784326-88554421-9b0d-4f2b-8267-9ca2a43d7a4d.png)

**Hint** 
We have to create an object of this ChromeOptions class and apply the addArguments method on it. The parameter -**-disable-notifications** is passed as a parameter to this method.


```
@BeforeMethod
    public void setUpBrowser() {
        ChromeOptions optionsChrome = new ChromeOptions();
        optionsChrome.addArguments("--incognito","--disable-notifications");
        driver = new ChromeDriver(optionsChrome);
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.manage().window().maximize();
    }

```



![image](https://user-images.githubusercontent.com/104761482/222973641-98a2d1e3-2676-44c2-b164-8b1c581371cb.png)